docs/pages/architecture.md

```diff
@@ -4,37 +4,50 @@ title: Meshery Architecture
 permalink: architecture
 ---
 
-# Architecture
+## Architecture
 
-<a href="{{site.baseurl}}/assets/img/architecture/meshery-architecture-diagram.svg"><img src="{{site.baseurl}}/assets/img/architecture/meshery-architecture-diagram.svg" /></a>
+<h4>The Meshery architecture can be observed in two perspectives:</h4> 
+ 
+##### 1. <a href="#clients"><b>Clients</b></a>
+##### 2. <a href="#providers"><b>Providers</b></a> 
 
-## Clients
+<br /><a href="{{site.baseurl}}/assets/img/architecture/meshery-architecture-diagram.svg"><img src="{{site.baseurl}}/assets/img/architecture/Meshery-architecture-diagram.png" /></a>
```

Comment:
Try not use use `<br>` if you can. 

If there's no special formatting that can't handled by markdown, let's try to avoid using `<a>` and `img` tags (html), but use markdown instead.